### PR TITLE
Change Case, Account, and Opportunity Ids to right ones so that tests pass

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: libs/SmartSync/SmartSyncTests/MetadataManagerTests.m [hunk 1]
```diff
@@ -44,12 +44,12 @@ @interface MetadataManagerTests : XCTestCase
 @implementation MetadataManagerTests
 
 static NSInteger const kRefreshInterval = 24 * 60 * 60 * 1000;
-static NSString* const kAccountOneId = @"001O000000m3UBiIAM";
-static NSString* const kAccountOneName = @"TEST ACCOUNT DO NOT TOUCH";
-static NSString* const kOpportunityOneId = @"006O0000006Q787IAC";
-static NSString* const kOpportunityOneName = @"TEST OPP DO NOT TOUCH";
-static NSString* const kCaseOneId = @"500O0000004Lnoa";
-static NSString* const kCaseOneName = @"00001010";
```
Comment (kchitalia) [inline libs/SmartSync/SmartSyncTests/MetadataManagerTests.m]: These Ids no longer exist for the sdktest@cs1.com user. I had to update them to use an exisiting Case, Account , and Opportunity.

File: libs/SmartSync/SmartSyncTests/MetadataManagerTests.m [hunk 2]
```diff
@@ -44,12 +44,12 @@ @interface MetadataManagerTests : XCTestCase
 @implementation MetadataManagerTests
 
 static NSInteger const kRefreshInterval = 24 * 60 * 60 * 1000;
-static NSString* const kAccountOneId = @"001O000000m3UBiIAM";
-static NSString* const kAccountOneName = @"TEST ACCOUNT DO NOT TOUCH";
```
Comment (bhariharan) [inline libs/SmartSync/SmartSyncTests/MetadataManagerTests.m]: What's wrong with these ones? They should be created and deleted by the test anyway.

Comment (bhariharan) [inline libs/SmartSync/SmartSyncTests/MetadataManagerTests.m]: Never mind. Just saw your comment.

**Comment by bhariharan** [comment]
:+1:

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**